### ISSUE TITLE
[feature request] replace deprecated arch and os labels

### ISSUE THREAD

**Issue by Congrool** [openyurtio/openyurt#869]
**What would you like to be added**:
replace `beta.kubernetes.io/os` and `beta.kubernetes.io/arch` with `kubernetes.io/os` and `kubernetes.io/arch` respectively.

**Why is this needed**:
Currently, OpenYurt components use labels of `beta.kubernetes.io/os` and `beta.kubernetes.io/arch` as nodeSelector to decide which node to deploy, such as yurt-tunnel-server. See [its nodeSelector](https://github.com/openyurtio/openyurt/blob/master/config/setup/yurt-tunnel-server.yaml#L193-L194).

As decribed in the kubernetes [official doc](https://kubernetes.io/docs/reference/labels-annotations-taints/#:~:text=beta.kubernetes.io,io/os%20instead.), both `beta.kubernetes.io/os` and `beta.kubernetes.io/arch` are deprecated.

**others**
/kind feature
/kind good-first-issue

**Comment by rambohe-ch** [comment]
@Congrool we need to consider how to maintain compatibility with old versions. Otherwise, OpenYurt components can not run if nodes do not have corresponding labels after components upgrade.

**Comment by Congrool** [comment]
Considering that we only ensure openyurt to run on kubernetes above 1.17, `beta.kubernetes.io/os` and `beta.kubernetes.io/arch` are deprecated since 1.14 and removed from 1.18, also the compatibility has been included by kubernetes itself, because when we use kubernetes above 1.17, both `beta.kubernets.io/os` and `kubernetes.io/os` will be added to the node like follows:
```bash
$ kubectl get node openyurt-e2e-test-worker --show-labels
NAME                       STATUS   ROLES    AGE     VERSION   LABELS
openyurt-e2e-test-worker   Ready    <none>   4m34s   v1.22.7   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/arch=amd64,kubernetes.io/hostname=openyurt-e2e-test-worker,kubernetes.io/os=linux,openyurt.io/is-edge-worker=true
```
Thus, we will never encounter the situation that a node has only the `beta.kubernetes.io/os` and `beta.kubernetes.io/arch` labels. So, I think there's no problem for this migration. @rambohe-ch

**Comment by rambohe-ch** [comment]
> 

@Congrool  Thanks for your reply, and it's a good news that no problem with migration.
Would you be able to take over this feature?

**Comment by rambohe-ch** [comment]
The issue was fixed by #958

**Comment by rambohe-ch** [comment]
Fixes by #969
